dotCMS/src/integration-test/java/com/dotcms/contenttype/business/ContentTypeDestroyAPIImplTest.java

```diff
@@ -291,4 +292,41 @@ public void Test_Destroy_Persona_Like_Content_Type() throws DotSecurityException
 
     }
 
+    /**
+     * Method to Test: {@link ContentTypeDestroyAPIImpl#relocateContentletsForDeletion(ContentType, ContentType)}
+     * Given Scenario: We create a Content Type with a random number or contentlets in it.
+     * Expected Result: We call relocate and all the contentlets should be moved to the target content type.
+     * @throws DotDataException
+     */
+    @Test
+    public void Test_Relocate_Contentlets() throws DotDataException {
+        final Random random = new Random();
+        for (int i = 0; i < 20; i++) {
+            TestContentRelocation(random.nextInt(19));
+        }
+    }
+
+    /**
+     * Method to Test: {@link ContentTypeDestroyAPIImpl#relocateContentletsForDeletion(ContentType, ContentType)}
+     * @param numOfContentlets
+     * @throws DotDataException
+     */
+    void TestContentRelocation(int numOfContentlets) throws DotDataException{
+        final Tuple2<ContentType,Set<String>> source = simpleContentTypeWithData(numOfContentlets);
```

Comment:
connections have to be handle even on test, since you are using dotConnect close the conns at the end

Reply:
done